# Pensamiento geométrico para visualizar y argumentar

# Clase 2

### Sectores circulares

El área de un sector circular es

$$ A= \dfrac{\theta R^2}{2} $$

El área de un sector circular es

$$ A= \dfrac{t\pi R^2}{360} $$

**Nota importante:** Aquí el ángulo $\theta$ se mide en radianes y el ángulo $t$ se mide en grados.

<img src="seg_circular1.png" align="middle" width="300">

El perímetro de un sector circular se mide como
$$ P=R(\theta+2) $$

El perímetro de un sector circular se mide como
$$ P=R\left(\dfrac{\pi t}{180}+2\right) $$

**Nota importante:** Aquí el ángulo $\theta$ se mide en radianes y el ángulo $t$ se mide en grados.

* ***Ejemplo:*** Calcular el área de un sector circular de radio 3 cm y ángulo de $30\,{}^\circ$.

<img src="seg_circular1.png" align="middle" width="300">

* Datos:
    * $R=3$
    * $t=30^\circ$
    * $\theta=\dfrac{\pi}{6}\ rad$
    * $A=?$

$$ 30^\circ=\dfrac{30\pi}{180}\ rad=\dfrac{\pi}{6}\ rad $$

* Fórmula:

$$ A=\dfrac{\theta R^2}{2} $$

* Sustitución:

$$ A=\dfrac{\frac{\pi}{6} (3)^2}{2} $$

$$ =\dfrac{\frac{\pi}{6} 9}{2} $$

$$ =\dfrac{\frac{9\pi}{6}}{2} $$

$$ =\dfrac{\frac{3\pi}{2}}{\frac{2}{1}} $$

$$ =\dfrac{3\pi}{4} $$

In [1]:
3π/4

2.356194490192345

El área es $\dfrac{3\pi}{4}\ \text{cm}^2=2.356\ \text{cm}^2$

* ***Ejemplo:*** Calcular el área y perímetro de un sector circular de radio 5 pulgadas (in) y ángulo de $\dfrac{3}{4}\pi\ \rm{rad}$.

* Datos:
    * $R=5$
    * $\theta=\dfrac{3}{4}\pi\ \rm{rad}$
    * $A=?$

* Fórmula:

$$ A=\dfrac{\theta R^2}{2} $$

* Sustitución:

$$ A=\dfrac{\frac{3}{4}\pi (5)^2}{2} $$

$$ =\dfrac{\frac{3}{4}\pi 25}{2} $$

$$ =\dfrac{\frac{3}{4}\pi \frac{25}{1}}{2} $$

$$ =\dfrac{\frac{75}{4}}{\frac{2}{1}}\pi $$

$$ =\dfrac{75}{8}\pi $$

El área es $\dfrac{75\pi}{8}\ \text{in}^2$

* ***Ejemplo:*** Calcular el perímetro de un sector circular de radio 5 pies (ft) y ángulo de $\dfrac{3}{4}\pi\ \rm{rad}$.

<img src="seg_circular1.png" align="middle" width="300">

* Datos:
    * $R=5$
    * $\theta=\dfrac{3}{4}\pi\ \rm{rad}$
    * $P=?$

* Fórmula:

$$ P=R(2+\theta) $$

* Sustitución:

$$ P=5\left(2+\dfrac{3}{4}\pi\right)=10+\dfrac{15}{4}\pi=21.78 $$

In [2]:
10+15π/4

21.780972450961723

El perímetro del sector circular es $\left(10+\dfrac{15}{4}\pi\right)\ \text{ft}=21.78\ \text{ft}$

* ***Ejemplo:*** Calcular el perímetro de un sector circular de radio 3 y ángulo de $30\,{}^\circ$.

* Datos:

* Fórmula:

* Sustitución:

<img src="seg_circular1.png" align="middle" width="200">

## Triángulo

<img src="triangulo.png" align="middle" width="300">

**¿Qué es un triángulo (también trigono)?**
* Una figura geométrica que tiene 3 ángulos.
* Una figura geométrica que tiene 3 vértices no colineales.
* Un *polígono* (figura geométrica que tiene muchos lados) que tiene 3 lados (o también llamados catetos). 

**Importante:** La suma de los ángulos internos de cualquier triángulo es un ángulo llano:

$$ A+B+C=180^\circ=\pi\ \rm{rad} $$

**¿Qué es el cateto opuesto?**

Imaginemos un cuarto que tiene forma de triángulo. Ponemos una linterna en el ángulo $A$. El *cateto opuesto* es la pared que se ilumina.

<img src="cateto_opuesto.png" align="middle" width="300">

Entonces, cuando escribimos un triángulo como en las dos figuras anteriores tenemos que:
* "$a$ es el cateto opuesto al ángulo $A$"  ó  "$A$ es el ángulo opuesto al cateto $a$"
* "$b$ es el cateto opuesto al ángulo $B$" ó "$B$ es el ángulo opuesto al cateto $b$"
* "$c$ es el cateto opuesto al ángulo $C$" ó "$C$ es el ángulo opuesto al cateto $c$"

### Tipos de triángulos

<img src="triangulo.png" align="middle" width="300">

i) **Equilátero:** Tres lados iguales ($a=b=c$) ó tres ángulos iguales ($A=B=C=60^\circ=\dfrac{\pi}{3}\ rad$)

ii) **Isósceles:** Dos lados iguales (p.ej. $a=c$) ó dos ángulos iguales (p.ej. $A=C$)

iii) **Escaleno:** Ningún lado igual ($a\neq b\neq c$) ó ningún ángulo igual ($A\neq B\neq C$)

<img src="tipos_triang1.png" align="middle" width="700">

i) **Rectángulo:** Uno de sus ángulos es recto (vale $90^\circ$ ó $\dfrac{\pi}{2}\ \rm{rad}$)

ii) **Oblicuángulo:** Ninguno de sus ángulos es recto

<img src="tipos_triang2.png" align="middle" width="500">

Noten que:
* Un triángulo rectángulo puede ser un triángulo isósceles.
* Un triángulo rectángulo puede ser un triángulo escaleno.
* Un triángulo rectángulo **NO** puede ser un triángulo equilátero.
* Un triángulo oblicuángulo puede ser un triángulo isósceles.
* Un triángulo oblicuángulo puede ser un triángulo escaleno.
* Un triángulo oblicuángulo puede ser un triángulo equilátero.

### Teorema de Tales y triángulos

<img src="tales.png" align="middle" width="300">

*Si tenemos tres rectas paralelas y estas cortan a dos rectas que no son paralelas se producen segmentos proporcionales:*

$$ \dfrac{\overline{AB}}{\overline{AC}}=\dfrac{\overline{DE}}{\overline{DF}} $$

También se cumple que

$$ \dfrac{\overline{AB}}{\overline{BC}}=\dfrac{\overline{DE}}{\overline{EF}} $$

### Semejanza

Dos cosas semejantes si son de la misma forma aunque tengan distinto tamaño.

<img src="minime.jpg" align="middle" width="300">

### Triángulos semejantes

Dos triángulos son semejantes si tienen la misma forma pero distinto tamaño. Dicho de otra forma, dos triángulos son semejantes si los triángulos tienen los mismo ángulos y los lados son proporcionales. La manera en que nos aseguramos que sean semejantes es:
* si tienen todos sus ángulos iguales entre sí
* si tienen dos ángulos iguales entre sí
* si tienen un ángulo igual entre sí y sus dos de sus lados son proporcionales con los lados correspondientes del otro triángulo
* si todos sus lados son proporcionales entre sí

### Ejemplos de semejanza de triángulos

* ***Ejemplo:***

<img src="triangulo.png" align="middle" width="300">

Pinto una línea paralela y genero dos triángulos:
* $\Delta ABC$
* $\Delta DBE$

<img src="triang_sem1.png" align="middle" width="400">

Los separo:

<img src="triang_sem2.png" align="middle" width="550">

**¿Qué podemos decir?**

$$ \dfrac{\overline{AB}}{\overline{BD}}=\dfrac{\overline{BC}}{\overline{BE}}=\dfrac{\overline{AC}}{\overline{DE}} $$

**¿Qué podemos decir también?**

$$ \dfrac{\overline{AC}}{\overline{AB}}=\dfrac{\overline{DE}}{\overline{BD}} $$

***Ejemplo:*** Pensemos que los triángulos semejantes son:

<img src="semejanza1.png" align="middle" width="500">

$$ \dfrac{x+1}{4}=\dfrac{12}{6} $$

$$ \dfrac{x+1}{4}=2 $$

$$ x+1=2\cdot4=8 $$

$$ x=7 $$

$$ \dfrac{y-1}{8}=\dfrac{6}{12} $$

$$ \dfrac{8}{y-1}=\dfrac{12}{6} $$

$$ \dfrac{8}{y-1}=2 $$

$$ 8=2(y-1) $$

$$ \dfrac{8}{2}=y-1 $$

$$ 4=y-1 $$

$$ 5=y $$